### ISSUE TITLE
Error immediately after app start if button pressed

### ISSUE THREAD

**Issue by AMurkin** [remyroy/CDDA-Game-Launcher#91]
* Desc: accidentally clicked button, i think [Launch game], at first second after window appears.
* V 1.3.8
* OS: Windows-10-10.0.14393 (64-bit)
* `<class 'UnboundLocalError'>`
* local variable 'main_window' referenced before assignment
```
  File "cddagl\ui.py", line 955, in restore_previous
  File "cddagl\ui.py", line 1217, in game_directory_changed
  File "cddagl\ui.py", line 1250, in update_version
```

**Comment by remyroy** [comment]
Thanks for the report. I'll see how to solve this issue.

**Comment by remyroy** [comment]
This is fixed with 60c908253d313af67f22dcb7c13f14c021310fea . It should be in the next release.
